server/src/core/workspace/resolvers/workspace-member.resolver.ts

```diff
@@ -9,6 +9,7 @@ import { CheckAbilities } from 'src/decorators/check-abilities.decorator';
 import {
   DeleteWorkspaceMemberAbilityHandler,
   ReadWorkspaceMemberAbilityHandler,
+  UpdateWorkspaceMemberAbilityHandler,
```

Comment:
This should be taken care of by the newly added eslint plugin unused imports, shouldn't it ?

Reply:
You're right, I have added it on the server side